core/package.json

```diff
@@ -41,7 +41,8 @@
     "test": "babel-node ./node_modules/nyc/bin/nyc.js mocha -- --recursive test",
     "test:watch": "npm run test -- --watch",
     "bundlesize": "npm run bundle && bundlesize",
-    "prepush": "../scripts/remove-test-prefixes.sh && npm test"
+    "prepush": "../scripts/remove-test-prefixes.sh && npm test",
```

Comment:
Please add docu:validate to prepush command.

Reply:
Done